arroba/tests/test_datastore_storage.py

```diff
@@ -390,6 +390,22 @@ def test_create_remote_blob_image_aspect_ratio(self):
         self.assertEqual(21, blob.width)
         self.assertEqual(12, blob.height)
 
+    def test_create_remote_blob_video_aspect_ratio(self):
+        image_bytes = Path(__file__).with_name('video.mp4').read_bytes()
+        cid = CID.decode('bafkreihjtnvgcwfv2owgyazgts2hzeofov2imfplnialwmknjha2v33p6a')
+        mock_get = MagicMock(return_value=requests_response(image_bytes))
+
+        blob = AtpRemoteBlob.get_or_create(url='http://my/blob.mp4', get_fn=mock_get)
+        mock_get.assert_called_with('http://my/blob.mp4', stream=True)
+        self.assertEqual({
+            '$type': 'blob',
+            'ref': cid,
+            'mimeType': 'video/mp4',
+            'size': 3294368,
```

Comment:
3.3MB, hmm, any chance we could find a smaller test video? Here's one that's 30KB: https://github.com/user-attachments/assets/68d8894d-65f4-4825-adb3-e53dbbfc304c